s3/pkg/datastore/yig/conf/yig.sql

```diff
@@ -43,19 +43,28 @@ DROP TABLE IF EXISTS `multiparts`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
```

Comment:
Is this tables still in the database used by s3?  IMHO, it's better to have a spereate database for yig backend.

Reply:
@sfzeng these tables are only used by yig backend. and yig backend has a separate db which is insulated from the one used by s3.

Reply:
OK, thanks！